core/src/main/java/com/google/errorprone/bugpatterns/checkreturnvalue/CanIgnoreReturnValueSuggester.java

```diff
@@ -76,19 +77,39 @@ public final class CanIgnoreReturnValueSuggester extends BugChecker implements M
   private static final String IMMUTABLE = "com.google.errorprone.annotations.Immutable";
   private static final String CRV = "com.google.errorprone.annotations.CheckReturnValue";
   private static final String CIRV = "com.google.errorprone.annotations.CanIgnoreReturnValue";
+  private static final String REFASTER_AFTER_TEMPLATE =
+      "com.google.errorprone.refaster.annotation.AfterTemplate";
 
   private static final Supplier<Type> PROTO_BUILDER =
       VisitorState.memoize(s -> s.getTypeFromString("com.google.protobuf.MessageLite.Builder"));
 
   private static final ImmutableSet<String> BANNED_METHOD_PREFIXES =
       ImmutableSet.of("get", "is", "has", "new", "clone", "copy");
 
+  private final ImmutableSet<String> additionalExemptingMethodAnnotations;
+
+  public CanIgnoreReturnValueSuggester(ErrorProneFlags errorProneFlags) {
+    this.additionalExemptingMethodAnnotations =
+        errorProneFlags
+            .getList("CanIgnoreReturnValue:ExemptingMethodAnnotations")
+            .map(ImmutableSet::copyOf)
+            .orElseGet(ImmutableSet::of);
+  }
+
   @Override
   public Description matchMethod(MethodTree methodTree, VisitorState state) {
     MethodSymbol methodSymbol = getSymbol(methodTree);
 
-    // if the method is already directly annotated w/ @CRV or @CIRV, bail out
-    if (hasAnnotation(methodSymbol, CRV, state) || hasAnnotation(methodSymbol, CIRV, state)) {
+    // If the method is already directly annotated w/ @CRV, @CIRV, or @AfterTemplate, bail out.
+    if (hasAnnotation(methodSymbol, CRV, state)
+        || hasAnnotation(methodSymbol, CIRV, state)
+        || hasAnnotation(methodSymbol, REFASTER_AFTER_TEMPLATE, state)) {
+      return Description.NO_MATCH;
+    }
+
+    // If the method is annotated with an annotation that is exempted, bail out.
+    if (additionalExemptingMethodAnnotations.stream()
+        .anyMatch(annotation -> ASTHelpers.hasAnnotation(methodTree, annotation, state))) {
```

Comment:
We can:
1. Statically import this method.
2. Use the `MethodSymbol`-accepting overload used above.
3. Use this `anyMatch` also for the cases mentioned above.